src/commands.ts

```diff
@@ -76,19 +76,58 @@ export function addCommands(
       await settings.set('columnOrder', order);
     }
   });
+  app.commands.addCommand(CommandIDs.showCreateEmpty, {
+    isToggleable: true,
+    isToggled: () => {
+      return settings.composite
+        .createEmptySection as ISettingsLayout['createEmptySection'];
+    },
+    label: trans.__('Show "Create Empty" Section'),
+    execute: async () => {
+      const createEmptySection = settings.composite
+        .createEmptySection as ISettingsLayout['createEmptySection'];
+      await settings.set('createEmptySection', !createEmptySection);
+    }
+  });
   app.commands.addCommand(CommandIDs.showStarred, {
     isToggleable: true,
     isToggled: () => {
       return settings.composite
         .starredSection as ISettingsLayout['starredSection'];
     },
-    label: trans.__('Show Starred Section'),
+    label: trans.__('Show "Starred" Section'),
     execute: async () => {
       const starredSection = settings.composite
         .starredSection as ISettingsLayout['starredSection'];
       await settings.set('starredSection', !starredSection);
     }
   });
+  app.commands.addCommand(CommandIDs.showNotebookLauncher, {
+    isToggleable: true,
+    isToggled: () => {
+      return settings.composite
+        .launchNotebookSection as ISettingsLayout['launchNotebookSection'];
+    },
+    label: trans.__('Show "Launch New Notebook" section'),
```

Comment:
Nitpick: only this one had lowercase "section", aligning it now:

```suggestion
    label: trans.__('Show "Launch New Notebook" Section'),
```